src/keybindings/KeybindingsPage.vala

```diff
@@ -218,7 +218,12 @@ namespace Ilia {
         }
 
         private void execute_keybinding (string exec) {
-            string commandline = "/usr/bin/i3-msg \"" + exec + "\"";
+            string cli_bin = get_wm_cli();
+
+            if(cli_bin == null) {
```

Comment:
nit: whitespace formatting

Reply:
suggestion: in case `get_wm_cli()` returns null lets emit an error message so it's clear to users something awry

Reply:
> suggestion: in case `get_wm_cli()` returns null lets emit an error message so it's clear to users something awry

Sure. Will do.